common/app/views/fragments/atoms/ampYoutube.scala.html

```diff
@@ -1,4 +1,5 @@
-@(media: model.content.MediaAtom,  displayCaption: Boolean, mainMedia: Boolean)(implicit request: RequestHeader)
+@import model.content.MediaWrapper
+@(media: model.content.MediaAtom,  displayCaption: Boolean, mediaWrapper: Option[MediaWrapper])(implicit request: RequestHeader)
```

Comment:
Looking at other files we are far from being consistent so feel free to ignore my comment I guess